TASK/SetPedCanPlayAmbientIdles.md

```diff
@@ -6,18 +6,14 @@ aliases: ["0x8FD89A6240813FD0"]
 
 ```c
 // 0x8FD89A6240813FD0 0xB5AD044E
-void SET_PED_CAN_PLAY_AMBIENT_IDLES(Ped ped, BOOL p1, BOOL p2);
+void SET_PED_CAN_PLAY_AMBIENT_IDLES(Ped ped, BOOL disableIdleAnims, BOOL stopCurrentAnim);
 ```
 
-```
-Appears only in fm_mission_controller and used only 3 times.  
-ped was always PLAYER_PED_ID()  
-p1 was always true  
-p2 was always true  
-```
+Prevents a ped from playing ambient idle animations.
+This native must be called every frame.
```

Comment:
Can you change the description in this format:

```
Prevents a ped from playing ambient idle animations.

Note: This native must be called every frame.
```

Reply:
To a codeblock?